eng/Versions.props

```diff
@@ -74,7 +74,7 @@
     <MicrosoftVisualStudioPackagesVersion>17.5.274-preview</MicrosoftVisualStudioPackagesVersion>
     <RoslynPackageVersion>4.6.0-3.23164.6</RoslynPackageVersion>
     <VisualStudioLanguageServerProtocolVersion>17.6.4-preview</VisualStudioLanguageServerProtocolVersion>
-    <MicrosoftNetCompilersToolsetVersion>4.6.0-2.23128.3</MicrosoftNetCompilersToolsetVersion>
+    <MicrosoftNetCompilersToolsetVersion>$(RoslynPackageVersion)</MicrosoftNetCompilersToolsetVersion>
```

Comment:
I'm not sure if this change should be conditioned for source-build only. It makes sense for `Microsoft.Net.Compilers.Toolset` package to be consumed from the same build as other `roslyn` packages, as it is built in `roslyn`. However, I don't know if this was an omission or intentional.

Reply:
I don't think it should be conditioned. I'm not sure why M.N.C.T should diverge from the rest of the roslyn depss. The versions currently in there suggests that maybe this was an oversight in the first place.

Reply:
I know in VS Mac we have at times had to split roslyn dependency for IDE tooling, from roslyn version used to build, but those times were few and far between, and because of strange mono/mac/arm issues. I would hope we don't hit anything that exciting here, so keeping them in sync makes sense to me.